docs/index.md

```diff
@@ -1,12 +1,24 @@
 # CCHDO Documentation
 
-Welcome, explore our resources and guides below: 
+Welcome to the CCHDO documentation site!
+
+This is the learning companion site to the CLIVAR and Carbon Hydrographic Data Office (CCHDO) found at [cchdo.ucsd.edu](https://cchdo.ucsd.edu).  
+For data downloads and cruise information, please visit [cchdo.ucsd.edu](https://cchdo.ucsd.edu).
+
+Information about file formats, our parameters, and data usage notebooks can be found under **[Getting Started](getting_started/index)**.  
```

Comment:
This is optional: but if you use the form `[](getting_started/index)` (leave the [] blank) it will always use the title of that page as the link text, so it will always match the top menu bar if the title changes.

Reply:
Thanks, that's good to know